### ISSUE TITLE
Vagrant build hangs on 'vagrant-mounted'

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#2242]
The following lines from `vagrant_mounted.pp`:

```puppet
...
    notify  => Service['workaround-vagrant-bug-6074'],
}

## start 'workaround-vagrant-bug-6074' service
service {'workaround-vagrant-bug-6074':
    ensure => 'running',
    enable => 'true',
...
```

or the corresponding `vagrant_mounted.erb` need to be adjusted.  Otherwise, the build will hang indefinitely, without any error exit statement.

Without adjusting the above logic, a manual fix can be implemented on each fix, to ensure the build resumes.  This manual fix involves running a separate terminal console, and manually `sudo service workaround-vagrant-bug-6074 stop`, which will cause the build to resume.  However, we need to ensure the above snippet doesn't run on initial build.

**Note:** the above problem has been [stated](https://github.com/mitchellh/vagrant/issues/6074#issuecomment-156530746) within the [vagrant](https://github.com/mitchellh/vagrant) repository.

**Comment by jeff1evesque** [comment]
If i revert `vagrant_mounted.pp` to the [version](https://github.com/jeff1evesque/machine-learning/blob/55b99874f085987fc2f4b513a7c6ab766e9017c2/puppet/manifests/vagrant_mounted.pp) included on [release 0.1](https://github.com/jeff1evesque/machine-learning/releases/tag/0.1), the build does not freeze.

**Comment by jeff1evesque** [comment]
9101970, 8a50c73: after these above commits, the build does not freeze.  The web-interface performs as expected, and the generic unit tests, defined in [`pytest_session.py`](https://github.com/jeff1evesque/machine-learning/blob/65702f034010802dcfbebdb3b18e2203f034b0e9/test/programmatic_interface/pytest_session.py) passes all four general tests.
